dropwizard-rabbitmq/src/main/java/smartthings/dw/rabbitmq/RabbitMQConfiguration.java

```diff
@@ -0,0 +1,91 @@
+package smartthings.dw.rabbitmq;
+
+import com.rabbitmq.client.ConnectionFactory;
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class RabbitMQConfiguration {
+
+    @NotEmpty
+    private String host;
+
+    @NotEmpty
+    private String virtualHost = "/";
+
+    @NotEmpty
+    private String username;
+
+    @NotEmpty
+    private String password;
+
+    private Integer port = null;
```

Comment:
Should we default this to the default rabbitmq port?

Reply:
I don't have a strong preference here, if the port isn't set in the ConnectionFactory it does to the default, see `RabbitMQConfigurationSpec`. If we want to more specific about the port I would be fine with that.